components/filesystem.rst

```diff
@@ -212,7 +212,8 @@ support symbolic links, a third boolean argument is available::
 
 :method:`Symfony\\Component\\Filesystem\\Filesystem::readlink` read links targets.
 
-PHP's ``readlink()`` function returns the target of a symbolic link. However, its behavior
+PHP's :method:`Symfony\\Component\\Filesystem\\Filesystem::readlink`
```

Comment:
Instead of this, we need to use the `:phpfunction:` directive to link to PHP functions (when used, don't add the parenthesis to the function name, they are added automatically):

```rst
PHP's :phpfunction:`readlink` function returns ...
```